src/sentry/api/endpoints/user_password.py

```diff
@@ -14,7 +14,7 @@ class UserPasswordSerializer(serializers.Serializer):
 
     def validate_password(self, value):
         user = self.context["user"]
-        if user.has_usable_password and not user.check_password(value):
+        if not user.check_password(value):
```

Comment:
Just checking, but this is safe because `check_password` already calls `has_usable_password`?

Reply:
Yeah.